### PR TITLE
SEV-SNP ACI: simplify primary container command

### PR DESCRIPTION
**Pull request by jumaffre**
This PR aims to fix some of the instabilities of the SNP pipeline by simplifying the container image's `command` and moving what is possible to the associated Dockerfile (`ccf_ci_built`) instead.

### CODE REVIEW

File: .azure-pipelines-templates/deploy_aci.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ jobs:
           echo "##vso[task.setvariable variable=gitSha;isOutput=true]$HEAD_SHA"
           until [ $RETRIES -eq 0 ] || [[ `az acr repository show-tags -n ccfmsrc --repository ccf/ci` == *"$HEAD_SHA"* ]]
           do
-            sleep 60
+            sleep 10
```
Comment (DomAyre) [inline .azure-pipelines-templates/deploy_aci.yml:60]: Ken asked us recently to change this to 60, he thought only waiting 10 seconds wasn't too far from DoS'ing ACR

**Review by DomAyre — APPROVED** [review]
LGTM if the pipeline passes

File: .azure_pipelines_snp.yml [hunk 2]
```diff
@@ -40,8 +40,7 @@ jobs:
       used_by:
         - test_snp
       secondaries:
-        image: ccfmsrc.azurecr.io/public/ccf/app/run:3.0.4-snp
-        count: 2
+        count: 0 # Disabled for now because of ACI instability
```
Comment (achamayou) [inline .azure_pipelines_snp.yml:43]:
```suggestion
        count: 0 # Disabled for now
```

File: scripts/azure_deployment/arm_aci.py [hunk 3]
```diff
@@ -489,13 +479,16 @@ def check_aci_deployment(
 
             while current_time < end_time:
                 try:
+                    print(
```
Comment (DomAyre) [inline scripts/azure_deployment/arm_aci.py:482]: For the sake of documentation, further improvement could be to use az container exec to setup keys in which case this check of ssh connection won't be necessary

**Review by DomAyre — APPROVED** [review]

**Pull request merged**